From: Jan Grulich <jgrulich@redhat.com>
Date: Tue, 4 Jul 2023 08:34:00 +0000
Subject: Bug 1839829 - WebRTC backport: PipeWire video capture - initialize
 pw_stream raw pointer member r=webrtc-reviewers,ng

This is a simple backport of an WebRTC upstream change.

Upstream commit: e21745a78b430ed4f2119b6342acbaa30a52b406

Differential Revision: https://phabricator.services.mozilla.com/D182671
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/2174ddbbb4733a8f9b9cef34d2e6c3a1ceb244d4
---
 modules/video_capture/linux/video_capture_pipewire.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/video_capture/linux/video_capture_pipewire.h b/modules/video_capture/linux/video_capture_pipewire.h
index 2cd6cca34b..01ff9c4ac7 100644
--- a/modules/video_capture/linux/video_capture_pipewire.h
+++ b/modules/video_capture/linux/video_capture_pipewire.h
@@ -48,7 +48,7 @@ class VideoCaptureModulePipeWire : public VideoCaptureImpl {
   VideoCaptureCapability configured_capability_;
   bool started_ RTC_GUARDED_BY(api_lock_);
 
-  struct pw_stream* stream_;
+  struct pw_stream* stream_ = nullptr;
   struct spa_hook stream_listener_;
 };
 }  // namespace videocapturemodule
-- 
2.37.3

